api/src/main/java/javax/mvc/engine/ViewEngineContext.java

```diff
@@ -74,21 +71,6 @@
     Models getModels();
 
     /**
-     * Returns HTTP request object from the Servlet container.
-     *
-     * @return HTTP request object.
-     */
-    HttpServletRequest getRequest();
-
-    /**
-     * Returns HTTP response object from the servlet container. The underlying
-     * output stream should be used to write the result of processing a view.
-     *
-     * @return HTTP response object.
-     */
-    HttpServletResponse getResponse();
-
```

Comment:
Maybe this affects also our discussion who is responsible for the character encoding of the response. Ozark currently exposes the bare `OutputStream` and a `PrintWriter` initialized with the charset found in `@Produces`. 

If the ViewEngine only has `setHeader(String,String)` it can't remove headers.